src/Infrastructure/Mapper/PropertyGroup/PropertyGroupNameMapper.php

```diff
@@ -25,15 +26,19 @@ public function map(
         Export $export,
         Shopware6PropertyGroup $shopware6PropertyGroup,
         AbstractAttribute $attribute,
-        ?Language $language = null
+        ?Language $language = null,
+        ?Shopware6Language $shopware6Language = null
     ): Shopware6PropertyGroup {
-        $name = $attribute->getLabel()->get($language ?: $channel->getDefaultLanguage());
+        $name = $attribute->getLabel()->get($channel->getDefaultLanguage());
```

Comment:
Same code as in PropertyGroupOptionNameMapper - try some refactor?

Reply:
Tutaj mimo, że kod wykonuje te same operacje to jednak działa na dwóch obiektach, których wolałbym nie łączyć do wspólnej logiki abstrakcyjnymi klasami - property group i property group option. Ewentualne wprowadzenie abstrakcji w tej klasie bedzie wymagalo wprowadzenia wspólnej abstrakcji również na modelach.